### ISSUE TITLE
bundle exec rake is failed with broken symlink

### ISSUE THREAD

**Issue by sofaray** [r7kamura/autodoc#46]
via https://github.com/r7kamura/autodoc/issues/44、broken symlink seem to be fixed.
but `Errno::ELOOP: Too many levels of symbolic links` is still happing.

**Comment by sanemat** [comment]
@r7kamura @ReoKuriyama @Danjie v.0.7.1 still has broken symlink.

```
$ curl -O https://rubygems.org/downloads/autodoc-0.7.1.gem
$ tar xvf autodoc-0.7.1.gem
$ tar xvzf data.tar.gz
$ ls -al spec/dummy/spec 
lrwxr-xr-x  1 murahashi.kenichi  86556269  21 Dec 18 14:05 spec/dummy/spec -> spec/dummy/../../spec
```

**Comment by sanemat** [comment]
I can't use bootsnap gem, because of this issue 🐰

**Comment by chendanjie** [comment]
I can't use bootsnap gem, because of this issue 🐰
+1

**Comment by r7kamura** [comment]
This is my mistake! I just re-released v0.7.2. Thank you for reporting 👍
